Dockerfile

```diff
@@ -71,6 +71,8 @@ RUN install-pip pipenv 2020.11.15
 
 # renovate: datasource=github-releases lookupName=python-poetry/poetry
 RUN install-tool poetry 1.1.11
+# Workaround until https://github.com/python-poetry/poetry/issues/4688 is fixed
+/usr/local/poetry/*/venv/bin/pip install cachecontrol==0.12.6
```

Comment:
This will need to be a `RUN` command. Does it need to be in the poetry venv? i.e. `RUN install-pip cachecontrol==0.12.6` won't work?

Reply:
I have no idea what `install-pip` does. But poetry has it's own virtual environment in the final container. So the downgrade of the package must be performed in that virtualenv.